package.json

```diff
@@ -17,11 +17,17 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "@emotion/react": "11.9.0",
+    "@emotion/styled": "11.8.1",
+    "@mui/icons-material": "5.6.2",
+    "@mui/material": "5.7.0",
+    "@mui/styled-engine-sc": "5.7.0",
     "axios": "0.21.4",
     "react": "17.0.0",
     "react-dom": "17.0.0",
     "react-query": "3.24.3",
-    "sass": "1.49.9"
+    "sass": "1.49.9",
```

Comment:
I see guys you are not using `sass` (so what adding it) and a few of you are not using even `css modules`. Is there some sort of miscommunication in the team?